src/d3a/models/area/__init__.py

```diff
@@ -68,13 +71,16 @@ def __init__(self, name: str = None, children: List["Area"] = None,
                  event_list=[],
                  grid_fee_percentage: float = None,
                  transfer_fee_const: float = None,
-                 external_connection_available=False):
+                 external_connection_available: bool = False,
+                 baseline_energy_settings=None):
```

Comment:
```suggestion
                 baseline_energy_settings: BaselineSettingsDefault  =None):
```

Reply:
Let me check whether with this change the user is able to set only one member of BaselineSettingsDefault. If so, I will change this. Thanks